### DISCUSSION TITLE
Fully Browser Settings

### DISCUSSION THREAD

**Discussion by Mephisto090** [Zefau/ioBroker.jarvis#2252] [Q&A]
Hej @Zefau,

du schreibst hier du benutzt auch den Fully Browser:
https://github.com/Zefau/ioBroker.jarvis/issues/2199#issuecomment-1416260413

Kannst du mir ggf. deine Einstellungen zur Verfügung stellen sodass es Perfekt für Jarvis eingerichtet ist=

VG

**Comment by Zefau** [comment]
Ich benutze fully nur zum Test bzw. zum Nachvollziehen von Fehlern. Da kann @mcuiobroker sonst sicher besser helfen.
